### DISCUSSION TITLE
Drag and drop in tree will open a new tab(firefox)

### DISCUSSION THREAD

**Discussion by Eve-Sama** [NG-ZORRO/ng-zorro-antd#5828] [Q&A]
### Reproduction link
[https://github.com/SortableJS/Sortable/issues/985](https://github.com/SortableJS/Sortable/issues/985)

### Steps to reproduce
en: In Firefox browser, drag and drop in `Tree` component will open a new tab. 

cn: 在火狐浏览器中, Tree组件的拖拽会引起快捷搜索(并非所有火狐都能复现, 我们的客户复现了, 大概就是拖拽组件的时候浏览器会突然新弹一个搜索页面, 搜索的内容是tree的key)

### What is expected?
en: Do not open a tab

cn: 不弹新的tab

### What is actually happening?
en: Seach with key of tree's node

cn: 会以tree的key为关键字进行搜索

| Environment | Info |
|---|---|
| ng-zorro-antd | 9.3.0 |
| Browser | Firefox |

**Comment by vthinkxie** [comment]
Hi @Mr-Eve 
can you submit a PR to fix this?
I have seen some comments here
https://github.com/SortableJS/Sortable/issues/985#issuecomment-282646581

**Comment by Eve-Sama** [comment]
Sure. I'm ready to do that. I need to open a issue before submit a PR. Thanks for your quick reaction.

**Comment by Eve-Sama** [comment]
I'm so sorry to report this problem. I can't run the project, the errors as below. Before run the project, I tried following steps:

- git fetch and based on latest master branch
- delete node_modules and use yarn install again

![image](https://user-images.githubusercontent.com/30228406/94132444-e8f05900-fe91-11ea-8261-18b413a0ac17.png)

**Comment by Eve-Sama** [comment]
Sometimes I can run the project. After several zorro's updates, run fails. And after several updates, I can run it successfully again. That's really weird.

**Comment by vthinkxie** [comment]
cc @hsuanxyz

**Comment by hsuanxyz** [comment]
try [An unhandled exception occurred: Cannot read property ‘Minus’ of undefined](https://stackoverflow.com/questions/63764017/an-unhandled-exception-occurred-cannot-read-property-minus-of-undefined) ?

**Comment by Eve-Sama** [comment]
> 
> 
> try [An unhandled exception occurred: Cannot read property ‘Minus’ of undefined](https://stackoverflow.com/questions/63764017/an-unhandled-exception-occurred-cannot-read-property-minus-of-undefined) ?

I have searched it already, but it doesn't work for me.

**Comment by Eve-Sama** [comment]
@hsuanxyz It looks like have been solved by close firefox's addon. You can find more details in ant-degisn's issue as up. You can try to fix this issue in zorro or close this issue. :D
